pkgs/tools/filesystems/wipefreespace/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, fetchurl
+, e2fsprogs
+, ntfs3g
+, xfsprogs
+, reiserfsprogs
+, reiser4progs
+, jfsutils
+, libuuid
+, texinfo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wipefreespace";
+  version = "2.5";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/wipefreespace/wipefreespace/${version}/wipefreespace-${version}.tar.gz";
+    sha256 = "sha256-wymV6G4Et5TCoIztZfdb3xuzjdBHFyB5OmI4EcsJKwQ=";
+  };
+
+  # missed: FAT12/16/32 MinixFS HFS+ OCFS
+  buildInputs = [
+    e2fsprogs
+    ntfs3g
+    xfsprogs
+    reiserfsprogs
+    reiser4progs
+    jfsutils
+    libuuid
+  ];
+
+  nativeBuildInputs = [
+    texinfo
+  ];
+
+  meta = with lib; {
+    description = "A program which will securely wipe the free space";
+    homepage = "https://wipefreespace.sourceforge.io";
+    license = licenses.gpl2Plus;
+    platforms = platforms.all;
```

Comment:
It currently doesn't evaluate on Darwin because multiple dependencies (`xfsprogs`, `reiserfsprogs`, `reiser4progs`, `jfsutils` and the missing `libaal`) are not supported. Even with all of them commented out, building fails:
![Bildschirmfoto von 2022-07-05 13-56-07](https://user-images.githubusercontent.com/23431373/177321961-988a89f5-153f-403a-a88e-cd0899d4305d.png)

There is this ~6 years old bug report about Darwin compatibility which also encountered "lots of errors" while compiling: https://sourceforge.net/p/wipefreespace/bugs/5/

Additionally the homepage, while not outright saying that it's Linux-only, only lists Linux distros in its compatibility section.
![Bildschirmfoto von 2022-07-05 13-55-50](https://user-images.githubusercontent.com/23431373/177322179-eff60cb4-8096-4e2e-bb0a-f4bc54716816.png)

I think marking this as Linux-only makes sense for now.
```suggestion
    platforms = platforms.linux;
```

Reply:
@OPNA2608 indeed, I never tried to build it on macOS